Update repository.md (#118)
Clinical-Genomics/development: docs/dev/repository.md

1. **Choose a branching model**:
The repository should state the branching model used for the repository in the `CONTRIBUTING` doc. Here is one way of describing a few popular [branching models](models.md). See AM doc #1042.

1. **Versioning**:
The repository should state the versioning scheme followed e.g. [semantic versioning](https://semver.org/) in the `CONTRIBUTING` doc. For example, bump version on each change on master, e.g. use the [bumpversion package](https://github.com/peritus/bumpversion) to automate bumping and tagging. This should follow the choosen branching model. Ensure that archived releases are made of each version you intend to deploy, and ideally all.

1. **Add a pull request template**: 
The repository should have a pull request template in `.github/PULL_REQUEST_TEMPLATE`, e.g [servers](https://github.com/Clinical-Genomics/servers/blob/master/.github/PULL_REQUEST_TEMPLATE), that includes:
    - Description of feature/fix
    - How to test and expected outcome
    - For version bumps, motivate according to your versioning scheme. E.g. in SemVer this is a major|minor|path version because ...

>     - Code review approval (if for some reason the code reviewer is different from the user approving the change)

Reviewer: ```suggestion
    - Code review approval
```

>     - Dito signoff tickboxes for code review, test status, and 

Reviewer: ```suggestion
    - Signoff tickboxes for code review, test status, and 
```

>     - If merging to a branch that will be deployed in production sign off on deployment, if for some reason the update deployment log is down. See [approving PRs](../publish/approving-prs.md). 

Reviewer: ```suggestion
    - If merging to a branch that will be deployed in production sign off on deployment, consider using a tickbox for deploy readiness
```

| Urgency S | Red | a82330 |
| Urgency M | Orange | ed851e |
| Urgency L |  Green | 4dc13c |

> Enhancement

Reviewer: ```suggestion
| Enhancement | Cyan blue | 84b6eb |
| Bug |  Magenta | d60493 |
| Refactor | Light green | 74f79b |
```
